util/shell_exec.go

```diff
@@ -2,38 +2,33 @@ package util
 
 import (
 	"io/ioutil"
+	"fmt"
 	"os"
 	"os/exec"
+	"strings"
 	"syscall"
 
 	"github.com/fatih/color"
 )
 
 const defaultFailedCode = 1
 
+type Executor struct {
+	cmd *exec.Cmd
+}
+
 // StreamCommand sets up the output streams (and colors) to stream command output if verbose is configured
 func StreamCommand(cmd *exec.Cmd) error {
-	return RunCommand(cmd, false)
+	return Executor{cmd}.Execute(false)
```

Comment:
You know, we could make this and `ForceStreamCommand` even easier to read on the caller side if we just take the command and args here instead of the `exec.Cmd` and call `exec.Command` just like the `Executor.Execute`